metadata/experiment.go

```diff
@@ -254,34 +266,19 @@ func readExperiment(bkt *bolt.Bucket, experiment *Experiment) error {
 	})
 }
 
-func readExperimentDefinition(bkt *bolt.Bucket) (ExperimentDefinition, error) {
+func readExperimentDefinition(bkt *bolt.Bucket, experiment *Experiment) (ExperimentDefinition, error) {
 	var edef ExperimentDefinition
-	/* TODO: enable in a followup PR
 	dbkt := bkt.Bucket(bucketKeyDefinition)
 	if dbkt == nil {
+		// TODO: should we error here
```

Comment:
No, validation is performed at write time. If our schema evolves, we want to read out what we can without erroring.

Reply:
makes sense